IPython/core/tests/test_application.py

```diff
@@ -33,15 +33,15 @@ def test_unicode_cwd():
 @dec.onlyif_unicode_paths
 def test_unicode_ipdir():
     """Check that IPython starts with non-ascii characters in the IP dir."""
-    ipdir = tempfile.mkdtemp(suffix=u"€")
+    ipdir = Path(tempfile.mkdtemp(suffix=u"€"))
     
     # Create the config file, so it tries to load it.
-    with open(os.path.join(ipdir, 'ipython_config.py'), "w") as f:
+    with ipdir.joinpath("ipython_config.py").open("w") as f:
         pass
```

Comment:
might be able to replace with something like https://docs.python.org/3/library/pathlib.html#pathlib.Path.touch

```diff
(ipdir / "ipython_config.py").touch()
```

Reply:
Good catch!